cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -863,9 +861,7 @@ pub mod bridging {
 			pub BridgeTable: sp_std::vec::Vec<NetworkExportTableItem> = sp_std::vec![
 				NetworkExportTableItem::new(
 					EthereumNetwork::get(),
-					Some(sp_std::vec![
-						EthereumLocation::get().interior.split_global().expect("invalid configuration for Ethereum").1,
-					]),
+					Some(sp_std::vec![Junctions::Here]),
```

Comment:
Is this right? 🤔

Reply:
Yeah, its basically the equivalent of 
```
EthereumLocation::get().interior.split_global().expect("invalid configuration for Ethereum").1
```

Because `EthereumLocation.get().interior` is `GlobalConsensus(Ethereum { chain_id: .. })`, then we `split_global()` which strips the first global consensus part, leaving `( GlobalConsensus(Ethereum { chain_id: .. }) , Junctions::Here )`. Then we take the second member of the tuple with `.1`.